lib/elixir/lib/exception.ex

```diff
@@ -997,6 +1017,20 @@ defmodule UndefinedFunctionError do
     ["      * ", Code.Identifier.inspect_as_function(fun), ?/, Integer.to_string(arity), ?\n]
   end
 
+  defp behaviours_for(module) do
+    attributes =
+      case function_exported?(module, :__info__, 1) do
+        true -> module.__info__(:attributes)
+        false -> module.module_info(:attributes)
```

Comment:
We don't need the conditional, we can just call `module.module_info(:attributes)` that should work for both.